conans/client/generators/cmake_common.py

```diff
@@ -583,7 +583,8 @@ class CMakeCommonMacros:
             # If using VS, verify toolset
             if (CONAN_COMPILER STREQUAL "Visual Studio")
                 if (CONAN_SETTINGS_COMPILER_TOOLSET MATCHES "LLVM" OR
-                    CONAN_SETTINGS_COMPILER_TOOLSET MATCHES "clang")
+                    CONAN_SETTINGS_COMPILER_TOOLSET MATCHES "clang" OR
+                    CONAN_SETTINGS_COMPILER_TOOLSET MATCHES "ClangCL")
```

Comment:
```suggestion
                    CONAN_SETTINGS_COMPILER_TOOLSET MATCHES "llvm" OR
                    CONAN_SETTINGS_COMPILER_TOOLSET MATCHES "clang" OR
                    CONAN_SETTINGS_COMPILER_TOOLSET MATCHES "Clang")
```

Reply:
The bug here is that MATCHES is case sensitive, so it would apply also to the "llvm" lowercase toolset, adding it too.